node/toolrunner.ts

```diff
@@ -584,27 +588,48 @@ export class ToolRunner extends events.EventEmitter {
                 this.pipeOutputToTool._getSpawnArgs(options),
                 this.pipeOutputToTool._getSpawnOptions(options));
 
+            let fileStream: fs.WriteStream = this.pipeOutputToFile ? fs.createWriteStream(this.pipeOutputToFile) : null;
+            if (fileStream) {
+                fileStream.write(this._getCommandString(options) + os.EOL);
+            }
+
             //pipe stdout of first tool to stdin of second tool
             cpFirst.stdout.on('data', (data: Buffer) => {
                 try {
+                    if (fileStream) {
+                        fileStream.write(data.toString());
```

Comment:
why `.toString()`?

Reply:
that's right, it should not be needed. I will pull it out and verify.